cves/CVE-2012-2774.yml

```diff
@@ -148,10 +155,10 @@ unit_tested:
     In FFmpeg, they have the FATE suite of regression tests. They are not
     "unit" tests per se, but are automated regression tests. If the fix for the
     vulnerability involved adding or updating a FATE suite, then make fix: true.
-  code:
-  code_answer:
-  fix:
-  fix_answer:
+  code: false
+  code_answer: There were no automated tests involved in the original code.
```

Comment:
Where did you look?

Reply:
The prompt suggested that I should be looking for commits related to the automated testing system in either the VCC or the vulnerability fixing commit. Neither of those commits (or the intervening ones) had any mention of automated tests, and both had corresponding user bug reports, so I assumed that both of my vulnerabilities (from 2012) predated automated unit testing of FFMPEG. I could be wrong.